app/api/v2/resource/documents.rb

```diff
@@ -38,6 +38,11 @@ class Documents < Grape::API
                    type: { value: Date, message: "resource.documents.expire_not_a_date" },
                    allow_blank: true,
                    desc: 'Document expiration date'
+          optional :doc_category,
```

Comment:
We added new field to `Document` model, so now we can understand if user submitted front page or selfie or back page of his doc